Tests/MSFT_xVMHyper-V/xVMHyper-V.Tests.ps1

```diff
@@ -64,6 +67,7 @@ Describe 'xVMHyper-V' {
         Mock -CommandName Get-VM -ParameterFilter { $Name -eq 'NonexistentVM' } -MockWith { Write-Error 'VM not found.'; }
         Mock -CommandName Get-VM -ParameterFilter { $Name -eq 'DuplicateVM' } -MockWith { return @([PSCustomObject] $stubVM, [PSCustomObject] $stubVM); }

```

Comment:
We should have a test to verify that get-targetresource accurately returns EnableGuestService